static/html/offline.html

```
@@ -4,114 +4,123 @@
+                <li>Cached interface styling and scripts</li>
+                <li>This fallback page while connectivity is restored</li>
             </ul>
         </div>
-        
-        <button class="btn btn-primary retry-btn" onclick="window.location.reload()">
-            <i class="bi bi-arrow-clockwise me-2"></i>Try Again
-        </button>
-    </div>
 
-    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/js/bootstrap.bundle.min.js"></script>
+        <button class="retry-btn" onclick="window.location.reload()">Try Again</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)